hack/make-rules/test-cmd-util.sh

```diff
@@ -357,6 +357,9 @@ run_pod_tests() {
   # Describe command (resource only) should print detailed information
   kube::test::describe_resource_assert pods "Name:" "Image:" "Node:" "Labels:" "Status:"
 
+  # Posting a pod to namespaces should fail.  Also tests --raw forcing the post location
```

Comment:
I think it worth to add a test cast to make sure a valid posting should succeed.